crates/bevy_core_pipeline/src/clear_color.rs

```diff
@@ -1,10 +1,10 @@
 use bevy_derive::{Deref, DerefMut};
 use bevy_ecs::prelude::*;
-use bevy_reflect::{Reflect, ReflectDeserialize, ReflectSerialize};
+use bevy_reflect::{FromReflect, Reflect, ReflectDeserialize, ReflectSerialize};
 use bevy_render::{color::Color, extract_resource::ExtractResource};
 use serde::{Deserialize, Serialize};
 
-#[derive(Reflect, Serialize, Deserialize, Clone, Debug, Default)]
+#[derive(Reflect, Serialize, Deserialize, Clone, Debug, Default, FromReflect)]
 #[reflect(Serialize, Deserialize)]
```

Comment:
Nit: Should we register `ReflectDefault` for `FromReflect` to use? Same for the rest of the changes.

Reply:
If `RefoectDefault` is registered using the attribute macro, it will take that as a hint to use `Default` to create the output type. This means ignored or missing fields do not need to exist or define a default function in order for it to succeed in constructing the type.